plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -105,10 +102,10 @@
 
     <!-- A "Connector" represents an endpoint by which requests are received
          and responses are returned. Documentation at :
-         Java HTTP Connector: /docs/config/http.html (blocking & non-blocking)
+         Java HTTP Connector: /docs/config/http.html
          Java AJP  Connector: /docs/config/ajp.html
          APR (HTTP/AJP) Connector: /docs/apr.html
-         Define a non-SSL HTTP/1.1 Connector on port 8080
+         Define a non-SSL/TLS HTTP/1.1 Connector on port 8080
     -->
     <Connector address="0.0.0.0" port="8080" protocol="org.apache.coyote.http11.Http11NioProtocol"
```

Comment:
I did a test with "org.apache.coyote.http11.Http11Nio2Protocol", it works fine.
An interesting post by Mark Thomas states (http://thread.gmane.org/gmane.comp.jakarta.tomcat.user/242253) :

> There is very little to choose between them. Implementing an NIO2 connector was the only way to find out if there were any noticeable performance benefits or not (there weren't). Very long term (maybe Tomcat 10 onwards) one of NIO and NIO2 may get dropped but I think we are several years away from even thinking about that. Right now, I'd say go with NIO. It has been around longer and should - in theory at least - be a little more stable.

Since we do not seem to have clear pros for Http11Nio2Protocol over Http11NioProtocol, I would follow the advice and go for keeping Http11NioProtocol. WDYT ?